src/ai/backend/agent/docker/agent.py

```diff
@@ -434,6 +434,14 @@ async def apply_network(self, cluster_info: ClusterInfo) -> None:
                         },
                     }
                 )
+        elif "alternate-bridge" in self.local_config["container"]:
```

Comment:
Please add non-null check here, as `config["container"]["alternate-bridge"]` value will always exist (either as a string or `None`).
```suggestion
        elif self.local_config["container"].get("alternate-bridge") is not None:
```

Reply:
Thank you. I applied it.